h/models/organization.py

```diff
@@ -13,6 +13,12 @@
 ORGANIZATION_LOGO_MAX_CHARS = 10000
 
 
+def _strip_xmlns(tag):
```

Comment:
I never know what to do with little helper functions like this??

Reply:
This is absolutely fine and follows conventions in the codebase. If testing the functionality through the module's public interface ends up being difficult then it would be worth either making it public or moving to a utility module.